src/SourceBuild/Arcade/eng/common/templates/steps/source-build-build-tarball.yml

```diff
@@ -115,7 +115,7 @@ steps:
       find artifacts/prebuilt-report/ -exec cp {} --parents -t ${targetFolder} \;
       find src/ -type f -name "*.binlog" -exec cp {} --parents -t ${targetFolder} \;
       find src/ -type f -name "*.log" -exec cp {} --parents -t ${targetFolder} \;
-      find test/*/*/*/*/*/testing-smoke*/logs -exec cp {} --parents -t ${targetFolder} \;
+      find test/*/*/*/*/*/*/testing-smoke*/logs -exec cp {} --parents -t ${targetFolder} \;
```

Comment:
Does `**` work here?  Maybe for a follow-up PR so we can get this in ASAP but it seems like it would cut down on issues if it does, I believe Michael also ran into this once.